.gitignore

```diff
@@ -46,6 +46,7 @@ tags
 /config.h
 /contrib/win32/openssh/LibreSSL
 /contrib/win32/openssh/ZLib
+/contrib/win32/openssh/libfido2
```

Comment:
I'm getting an error while generating the ecdsa-sk. It didn't prompt me to touch the yubikey. Although it didn't ask, I keep touching the yubikey and running the same command but it still fails.

![image](https://user-images.githubusercontent.com/23668037/145897200-e0627cad-78b0-4a4e-a242-d63413e88f5b.png)

Reply:
fyi, I'm compiling using the latest commit on my windows 10 machine.

![image](https://user-images.githubusercontent.com/23668037/145897686-e943585a-ad43-4f41-b672-4c1ab00e699e.png)

Reply:
Looks like a build problem. Are you able to run ssh-sk-helper.exe as a standalone process and converse with it?

```
> .\ssh-sk-helper.exe -vvv
debug3: w32_fcntl ERROR: bad fd: 1
debug3: ssh_msg_recv entering
foo
ssh_msg_recv: read: bad msg_len 1718578954
ssh_msg_recv failed
```

If that doesn't work, please try rebuilding with Win32-OpenSSH.sln from ebc802a36, which works locally for me. If running ssh-sk-helper.exe works but ssh-keygen.exe still fails, can you please retry with debugging enabled?

```
> $Env:FIDO_DEBUG = 1
> .\ssh-keygen -vvv -t ecdsa-sk
```

Thank you,

-p.